zstd_vendor/CMakeLists.txt

```diff
@@ -5,13 +5,13 @@ find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_vendor_package REQUIRED)
 
 list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
-# We need at least VERSION 1.4.8, version check is done in Findzstd.cmake
-find_package(zstd 1.4.8 QUIET)
+# We need at least VERSION 1.5.6, version check is done in Findzstd.cmake
+find_package(zstd 1.5.6 QUIET)
```

Comment:
I think we should vendor version 1.5.5, as that is the version that will be shipped in Ubuntu 24.04: https://packages.ubuntu.com/noble/libzstd1